src/App.js

```diff
@@ -1,16 +1,15 @@
 import React, { Component } from "react";
-import Routes from "./routes/index";
+// import Routes from "./routes/index";
 import "font-awesome/css/font-awesome.min.css";
+import CourseForm from './containers/Dashboard/Forms/CourseForm';
 import "./App.css";
 import "./myOwn.css";
-// import { getItem } from "./services/Session";
-// import Navbar from "./containers/Dashboard/Navbar";
-// import Header from "./components/Header";
-// import { LOGIN } from "./constants/index";
+
 
 class App extends Component {
   render() {
-    return <Routes />;
+  return <CourseForm/>
```

Comment:
@Humnaa 
Create a separate route for forms e.g. `/course-form` where you show this specific form instead of replacing the actual `routes` in `App.js`.

Reply:
I have made separate routes for courseForm with `/course-form`